README.md

```diff
@@ -4,7 +4,9 @@ This repo contains Terraform modules to use for Firezone deployments on AWS.
 
 ## Examples
 
-- [NAT Gateway](./examples/nat-gateway): This example shows how to deploy one or
-  more Firezone Gateways in a single AWS VPC that is configured with a NAT
-  Gateway for egress. Read this if you're looking to deploy Firezone Gateways
-  behind a single, shared static IP address on AWS.
+- [Internet Gateway](./examples/internet-gateway): This example shows how to
+  deploy one or more Firezone Gateways in a single AWS VPC that is configured
+  with an Internet Gateway for egress. The Gateways will be associated to
+  dedicated Elastic IP Resources to minimize IP churn. Read this if you're
+  looking to deploy Firezone Gateways to AWS that need to communicate with the
+  internet using static IPs.
```

Comment:
Just want to make sure I'm understanding the intent here.  Are we saying that we are now requiring people to deploy their Firezone Gateway in an AWS VPC Public Subnet?

Reply:
I updated the module to take a bool var `attach_public_subnet`. This example is for the Internet Gateway so I think most folks will expect it to be attached to a public subnet, but the module won't require it.